easybuild/easyconfigs/f/FPM/FPM-1.4.0-Ruby-2.7.1.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'RubyGem'
+
+name = 'FPM'
+version = '1.4.0'
```

Comment:
@alirezaghavaminia Why not update to the latest available version (1.11.0)? See https://github.com/jordansissel/fpm/releases

Reply:
Oh, I see. I updated based on [RubyGems.org](https://rubygems.org/gems/fpm/versions/1.4.0) website.

Reply:
I think I had dependency conflict issues. I'm not sure...